cypress/integration/pages/mediaAssetPage/testsForCanonicalOnly.js

```diff
@@ -0,0 +1,34 @@
+import envConfig from '../../../support/config/envs';
+
+// For testing important features that differ between services, e.g. Timestamps.
+// We recommend using inline conditional logic to limit tests to services which differ.
+export const testsThatAlwaysRunForCanonicalOnly = ({ service, pageType }) => {
+  describe(`No testsToAlwaysRunForCanonicalOnly to run for ${service} ${pageType}`, () => {});
+};
+
+// For testing feastures that may differ across services but share a common logic e.g. translated strings.
+export const testsThatFollowSmokeTestConfigForCanonicalOnly = ({
+  service,
+  pageType,
+}) =>
+  describe(`Canonical Tests for ${service} ${pageType}`, () => {
+    // TODO Chartbeat not yet implemented
+    describe('Chartbeat', () => {
+      if (envConfig.chartbeatEnabled) {
+        it.skip('should have a script with src value set to chartbeat source', () => {
```

Comment:
Why are we adding this at all if it's skipped? Also this logic for Chartbeat doesn't seem page specific.